.github/scripts/install-slices/install_slices.py

```diff
@@ -319,6 +369,11 @@ def main() -> None:
         return
     # Install the slices in each package.
     for pkg in packages:
+        # Keep track of whether the copyright file is installed on every "cut"
+        # This should always be the case, whether enforced by a global "essential"
+        # or Chisel itself. Exception: the copyright file will not be installed
+        # if it doesn't exist in the deb itself.
+        installed_copyright = {}
```

Comment:
Sorry for not finding this in earlier reviews but my Python was a bit rusty, shouldn't `installed_copyright` be a set? This way we can check membership using `in` and check for emptyness below instead of `not all(installed_copyright.values())`.

Reply:
could, but then you wouldn't been able to identify which slices failed to install it. The CI installs slice by slice (not all in one go). So having it as a dict allows us to have the log message below, saying which slices are the culprit

Reply:
You are right! I was thinking a set of slices that did not install copyright (sorry to go back there again), then we can check for len() > 0 and all of the keys are faulty and can be reported.

Reply:
also done in 0bf534c